### PR TITLE
Replaces larval growth sting with neurotoxin sting, removes larval accelerant from the game

### PR DESCRIPTION
**Pull request by private-tristan**
## About The Pull Request

see title 

## Why It's Good For The Game

larval growth sting purged meds 3 times the rate of neurotoxin, now it's neurotoxin sting, which purges meds at 1/3rd the rate  but  drains stamina and can kill.

## Changelog
:cl:
balance: replaced shrike and drone's larval sting with neurotoxin sting.
del: removed larval accelerant from the game.
/:cl:

### CODE REVIEW

**Review by BraveMole — CHANGES_REQUESTED** [review]
Just remove larval and replace it by neurotox

**Comment by Losenis** [comment]
>People keep complaining that marine meds are too strong
>Let's nerf the ways xenos have to purge them

**Comment by Araulius** [comment]
>nerfing the counter to b18 and premedders

are you stupid? you have oodles of pills and injectors. what a non-issue.

**Comment by private-tristan** [comment]
alright, I'll see if I can replace it with neurotoxin.

**Review by BraveMole — CHANGES_REQUESTED** [review]
remove larval from the game, it's no longer used by anything

**Comment by Losenis** [comment]
It's no longer used by anything because you fucks keep removing it from everything, as queen screech + larval was my go-to tactic but you fucking people canned it due to some wild thing like too many buttons or whatever else and now claim nothing uses it

**Comment by BraveMole** [comment]
> It's no longer used by anything because you fucks keep removing it from everything, as queen screech + larval was my go-to tactic but you fucking people canned it due to some wild thing like too many buttons or whatever else and now claim nothing uses it
> 
> I see zero point for this PR other than nerfing for the sake of nerfing. Larval could already work as "purges meds hard as hell but doesn't actually damage/kill you" alternative. Not even could, already DOES.

neurotox is not worse than larval.

**Comment by BraveMole** [comment]
> It's no longer used by anything because you fucks keep removing it from everything, as queen screech + larval was my go-to tactic but you fucking people canned it due to some wild thing like too many buttons or whatever else and now claim nothing uses it
> 
> I see zero point for this PR other than nerfing for the sake of nerfing. Larval could already work as "purges meds hard as hell but doesn't actually damage/kill you" alternative. Not even could, already DOES.

Also if you want to add queen injection back, i don't think it's an issue. Especially with neuro

**Comment by private-tristan** [comment]
okay, I'll go delete larval from the game

**Comment by Losenis** [comment]
Neurotox is actually worse than larval in very short engagements because the premeds will still be doing something for a while longer than larval due to the far lower purge rate, and needs stam to drop to zero before even starting to do oxy and tox damage. If the guy is running paracetamol and ryetalin neurotox is basically useless since its purgerate is peanuts to do much in those few cycles. Defiler already has melee neurotox delivery, keep something else for the other xenos.

**Comment by private-tristan** [comment]
I THINK that was all mentions of larval accelerant in the code. not sure though because forks don't allow searching.

**Comment by private-tristan** [comment]
Okay, all checks passed!

**Comment by usnpeepoo** [comment]
why are you letting xenos larval inject you in the first place? this should be left alone

**Comment by novaepee** [comment]
Larval growth sting is a relic of capture gameplay. Like other relics of capture gameplay, they hit marines harder when spawners and silos replace larba burst. Carrier, disarm, eggs, defiler (remember when defiler puts an actual larva inside you?), and now larcval growth sting are subject to be corrected for the silo era.

The future is now.

**Comment by novaepee** [comment]
> > It's no longer used by anything because you fucks keep removing it from everything, as queen screech + larval was my go-to tactic but you fucking people canned it due to some wild thing like too many buttons or whatever else and now claim nothing uses it
> > I see zero point for this PR other than nerfing for the sake of nerfing. Larval could already work as "purges meds hard as hell but doesn't actually damage/kill you" alternative. Not even could, already DOES.
> 
> Also if you want to add queen injection back, i don't think it's an issue. Especially with neuro

I can do that after this PR get merged if Tristan doesn't do it.

**Comment by Waselon** [comment]
Next step: Remove lavral huggers

File: code/game/objects/machinery/vending/vending_types.dm [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 */
 
-/*
+/*w
```
Comment (BraveMole) [inline code/game/objects/machinery/vending/vending_types.dm:20]: small mistake

**Review by BraveMole — CHANGES_REQUESTED** [review]
rest looks good

File: code/game/objects/machinery/vending/vending_types.dm (hunk 1 above)
Comment (private-tristan) [reply code/game/objects/machinery/vending/vending_types.dm]: fixed.

**Review by BraveMole — APPROVED** [review]

**Pull request merged**